packages/components/src/focal-point-picker/stories/index.js

```diff
@@ -44,3 +44,33 @@ export const video = () => {
 
 	return <Example url={ url } />;
 };
+
+export const snapping = () => {
+	const snapValues = {
+		x: [ 0, 0.33, 0.66, 1 ],
+		y: [ 0, 0.33, 0.66, 1 ],
+	};
+
+	const thereshold = 0.05;
```

Comment:
Nit: typo `thereshold` instead of `threshold`

Reply:
Fixed